manifests/dev/args.yaml

```diff
@@ -1,10 +1,11 @@
 # Override args for development mode.
-apiVersion: apps/v1beta1
-kind: Deployment
+apiVersion: apps/v1beta2
+kind: StatefulSet
 metadata:
   name: metacontroller
   namespace: metacontroller
 spec:
+  serviceName: ""
```

Comment:
Since `args.yaml` and `image.yaml` contain partial objects, used for kustomize overlays, we don't need to mention fields here if the value is the same as the base manifest.